src/tasks/processBulkUpload.ts

```diff
@@ -301,6 +318,10 @@ const createProposalFieldValueForBulkUploadCsvRecord = async (
   return proposalFieldValue;
 };
 
+const getProcessedKey = (unprocessedKey: string): string => (
+  `${S3_PROCESSED_BULK_UPLOADS_KEY_PREFIX}/${path.basename(unprocessedKey)}`
```

Comment:
Might I instead suggest that the processed key should be `${S3_PROCESSED_BULK_UPLOADS_KEY_PREFIX}/${bulkUpload.id)}`? That'll make it easier for us to examine uploads manually, rather than requiring database lookups!

Reply:
I'm realizing that I had mentally assumed that the `processed` key might some day be the prefix for other types of file uploads -- which is probably a bad idea!

Which is just to say, I'm gonna update the key from being `processed/` to `processedBulkUploads/` at which point yes I think that `id` change will make sense and be safe from collisions.

Reply:
(oh, it was already `processedBulkUploads` lol)

Adding the id change!